oeps/oep-0010.rst

```diff
@@ -0,0 +1,202 @@
+=========================
+OEP-10: Open edX Releases
+=========================
+
++---------------+-------------------------------------------+
+| OEP           | :doc:`OEP-10 <oep-0010>`                  |
++---------------+-------------------------------------------+
+| Title         | Open edX Releases                         |
++---------------+-------------------------------------------+
+| Last Modified | 2016-10-14                                |
++---------------+-------------------------------------------+
+| Author        | Ned Batchelder, ned@edx.org               |
++---------------+-------------------------------------------+
+| Arbiter       | Edward Fagin                              |
++---------------+-------------------------------------------+
+| Status        | Draft                                     |
++---------------+-------------------------------------------+
+| Type          | Process                                   |
++---------------+-------------------------------------------+
+| Created       | 2016-10-14                                |
++---------------+-------------------------------------------+
+| `Resolution`  | <links to any discussions where the final |
+|               | status was decided>                       |
++---------------+-------------------------------------------+
+
+
+..
+    - Expectations for component owners
+
+
+Abstract
+========
+
+Open edX is packaged occasionally into releases. This document details the
+process.
+
+
+Motivation
+==========
+
+Open edX releases happen infrequently, every four to six months. Edx engineers
+are typically focused on the weekly edx.org releases.  This document
+standardizes aspects of the release process to ensure that all involved
+understand and participate appropriately.
+
+
+Specification
+=============
+
+
+Components
+----------
+
+When talking about software that is part of Open edX, there are a number of
+components that might be useful to discuss, of various sizes:
+
+- Application: this is a user-visible application that provides some useful
+  functionality.  Examples are the LMS, Studio, Analytics Insights.  These can
+  be composed of a number of different repos and services.
+
+- Service (or IDA): this is a program that can be separately installed and
+  started.  Often applications will need a number of these working together to
+  provide all functionality.  The LMS and Studio applications are implemented
+  by the edxapp service, which also uses the forums service.  A service might
+  or might not have a user interface.
+
+- Feature: large applications have major features which are configurable, and
+  should be called out explicitly if their support status is different than the
+  application they are a part of.
+
+
+
+Levels of support
+-----------------
+
+Units in Open edX are either either *included* or *supported* in a release.
+
+To be **included** in an Open edX release, a component, facility or feature
```

Comment:
This should just be "component," since that term is defined above to include a variety of things.